internal/driver/driver.go

```diff
@@ -562,6 +617,15 @@ func (d *Driver) newDevice(name string, protocols map[string]models.ProtocolProp
 		Scheme: RtspUriScheme,
 		Host:   fmt.Sprintf("%s:%s", d.rtspHostName, d.rtspTcpPort),
 	}
+	credential, edgexErr := d.tryGetCredentials(rtspAuthKey)
+	if edgexErr != nil {
+		d.lc.Warnf("failed to get credentials for at path %s", rtspAuthKey)
```

Comment:
reword. something like so:
```suggestion
		d.lc.Warnf("Failed to get credentials for rtsp authentication from secretName %s", rtspAuthKey)
```